scene/2d/position_2d.cpp

```diff
@@ -38,8 +38,9 @@ const float DEFAULT_GIZMO_EXTENTS = 10.0;
 void Position2D::_draw_cross() {
 
 	float extents = get_gizmo_extents();
-	draw_line(Point2(-extents, 0), Point2(+extents, 0), Color(1, 0.5, 0.5));
-	draw_line(Point2(0, -extents), Point2(0, +extents), Color(0.5, 1, 0.5));
+	// Colors taken from `axis_x_color` and `axis_y_color` (defined in `editor/editor_themes.cpp`)
```

Comment:
Since Position2D doesn't inherit from Control, I can't use `get_color()` here. What would be a good way to get the color without conditionally including files from `editor/`?